src/main/java/run/halo/app/model/properties/CommentProperties.java

```diff
@@ -14,6 +14,11 @@
      */
     GRAVATAR_DEFAULT("comment_gravatar_default", String.class, ""),
 
+    /**
+     * The identicon gravatar type.
+     */
+    GRAVATAR_IDENTICON("comment_gravatar_identicon", String.class, "identicon"),
```

Comment:
I don’t think it’s necessary, just use `GRAVATAR_DEFAULT` directly

Reply:
Fixed.